core/common/src/TimeZone.kt

```diff
@@ -33,7 +33,6 @@ import kotlin.time.Instant
  *
  * @sample kotlinx.datetime.test.samples.TimeZoneSamples.usage
  */
-@Serializable(with = TimeZoneSerializer::class)
 public expect open class TimeZone {
```

Comment:
Note that the moment you remove `@Serializable(with=...)` annotation from type, it is shown as red with SERIALIZER_NOT_FOUND where it was used in any `@Serializable` user class. Sadly, we do not have any kind of deprecation migration for that; so I suggest raising all `@Deprecated` here to ERROR just to speedup this process